lib/wallaroo/core/checkpoint/checkpoint_initiator.pony

```diff
@@ -46,6 +46,8 @@ actor CheckpointInitiator is Initializable
   let _event_log: EventLog
   let _barrier_initiator: BarrierInitiator
 
+  var _recovery: (Recovery | None) = None
```

Comment:
It doesn't seem like this is used.  Can we remove it, and the related behavior below?